common/source/docs/common-optical-flow-sensor-setup.rst

```diff
@@ -17,6 +17,9 @@ With the sensor connected to the autopilot, connect to the autopilot with the Mi
 Calibrating the sensor
 ======================
 #. Connect to your autopilot and ensure that logging while disarmed is enabled by setting :ref:`LOG_DISARMED <copter:LOG_DISARMED>` to 1
+[site wiki='rover']
+#. Ensure the :ref:`FLOW_HGT_OVR <rover:FLOW_HGT_OVR>` parameter is set to the height of the sensor above ground
```

Comment:
does this need to be changed to eye level when calibrating since that the height it gets cal'd at?

Reply:
and these errors are being generated besides that"/home/henry/Desktop/ardupilot_wiki/rover/source/docs/common-optical-flow-sensor-setup.rst:27: ERROR: Unexpected indentation.
/home/henry/Desktop/ardupilot_wiki/rover/source/docs/common-optical-flow-sensor-setup.rst:30: WARNING: Block quote ends without a blank line; unexpected unindent.
/home/henry/Desktop/ardupilot_wiki/rover/source/docs/common-optical-flow-sensor-setup.rst:65: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/home/henry/Desktop/ardupilot_wiki/rover/source/docs/common-optical-flow-sensor-setup.rst:119: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/home/henry/Desktop/ardupilot_wiki/rover/source/docs/common-optical-flow-sensor-setup.rst:216: WARNING: Block quote ends without a blank line; unexpected unindent.

Reply:
@stephendade  are you going to address these or do you want me to when I get time?

Reply:
I'll get to it in the next day or two

Reply:
> does this need to be changed to eye level when calibrating since that the height it gets cal'd at?

User should keep the vehicle at ground level and rotate roll/pitch from there, by lifting 1 side of the vehicle at a time.